must-call-checker/tests/mustcall/MustCallChoiceImpl.java

```diff
@@ -0,0 +1,22 @@
+// A simple test that the extra obligations that MustCallChoice imposes are
+// respected.
+
+// @skip-test until the checks are implemented
+
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.checker.objectconstruction.qual.*;
+import java.io.*;
+
+public class MustCallChoiceImpl implements Closeable {
```

Comment:
It would be good to write a version of this test also for the OCC.  For that I think we'd need to add a `@MustCall` annotation on the type and `@EnsuresCalledMethods` on `close()`